mne_qt_browser/_pg_figure.py

```diff
@@ -4709,7 +4719,7 @@ def closeEvent(self, event):
                         _disconnect(vl.sigPositionChangeFinished)
                     self.mne.vline.clear()
                 else:
-                    _disconnect(self.mne.vline.sigPositionChangeFinished)
+                    _disconnect(self.mne.vline.sigPositionChangeFinished, allow_error=True)
```

Comment:
I couldn't reproduce the CI-error locally so I assumed this is the way to go then